### ISSUE TITLE
Jira backend ?

### ISSUE THREAD

**Issue by yisraeldov** [arbox/org-sync#39]
A Jira backend would be really great, at work they use jira and I get used to writing my tasks in orgmod and syncying up would make life much easier.

**Comment by arbox** [comment]
There is a working backend focused on Jira. Try `org-jira`, a comparison would be very interesting.

**Comment by yisraeldov** [comment]
@arbox I don't see a working backend  for jira ( I wish there was ) . I am trying `org-jira` and it is more intended to be an interface to jira. You need to create tasks one at a time. Also there is a bug that doesn't let you close tasks so not so useful.

**Comment by yisraeldov** [comment]
@arbox  Just searched the code and found no mention of jira is it part of a third party package ?

**Comment by arbox** [comment]
Did you look at this: https://melpa.org/#/org-jira

**Comment by yisraeldov** [comment]
@arbox yes, that is what I talked about above, it is NOT a backed  for org-sync.
